react/src/components/Chat/CustomModelForm.tsx

```diff
@@ -51,64 +56,56 @@ const CustomModelForm: React.FC<CustomModelFormProps> = ({
         key={baseURL}
         initialValues={{
           baseURL: baseURL,
-          token: token,
+          token: undefined,
         }}
       >
-        {alert ? (
-          <div style={{ marginBottom: themeToken.size }}>{alert}</div>
-        ) : null}
+        <div style={{ marginBottom: themeToken.size }}>
+          <Alert
+            type="warning"
+            showIcon
+            message={t('chatui.CannotFindModel')}
+          />
+        </div>
         <Form.Item
-          label="baseURL"
-          name="baseURL"
+          label="Base Path"
+          name="basePath"
           rules={[
-            {
-              type: 'url',
-            },
             {
               required: true,
             },
           ]}
         >
-          <Input placeholder="https://domain/v1" />
-        </Form.Item>
-        <Form.Item
-          label="Model ID"
-          name="modelId"
-          rules={[
-            {
-              required: true,
-            },
-          ]}
-        >
-          <Input placeholder="llm-model" />
+          <Input
+            placeholder="v1"
+            addonBefore={origin}
+            defaultValue={pathname}
+          />
         </Form.Item>
         <Form.Item label="Token" name="token">
-          <Input />
+          <EndpointTokenSelect
+            loading={isPendingSubmit}
+            endpointId={endpointId}
+          />
         </Form.Item>
-      </Form>
-    </Flex>
-  );
-};
-
-type CustomModelAlertProp = {
-  onClick?: ButtonProps['onClick'];
-};
-
-const CustomModelAlert: React.FC<CustomModelAlertProp> = ({ onClick }) => {
-  const { t } = useTranslation();
-
-  return (
-    <Alert
-      type="warning"
-      showIcon
-      message={t('chatui.CannotFindModel')}
-      action={
-        <Button icon={<ReloadOutlined />} onClick={onClick}>
+        <Button
+          icon={<ReloadOutlined />}
+          loading={isPendingSubmit}
+          onClick={() => {
+            startSubmitTransition(() => {
+              const basePath =
+                formRef.current?.getFieldValue('basePath') ?? 'v1';
+              onSubmit?.({
+                baseURL: new URL(basePath, origin).href,
+                token: formRef.current?.getFieldValue('token'),
+              });
+            });
+          }}
+        >
           {t('button.Refresh')}
```

Comment:
We need to change the button title because of the change in the `chatui.CannotFindModel` message.

Reply:
I don't get it. Please tell me how to change the name.

Reply:
![Screenshot 2025-04-21 at 12.30.07 PM.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/JFysNkoR8J15CnzOUuvH/c2714089-a9dd-479f-a8eb-9a240f42f929.png)

![Screenshot 2025-04-21 at 12.29.22 PM.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/JFysNkoR8J15CnzOUuvH/9b1d5c8d-526c-4045-b0d2-6598cb1c065b.png)